test/chainrules.jl

```diff
@@ -13,6 +13,14 @@
         end
     end
 
+    for x in -2.0:2.0
+        test_scalar(xexpx, x)
+        for y in -2.0:2.0
+            test_frule(xexpy, x, y)
+            test_rrule(xexpy, x, y)
+        end
+    end
```

Comment:
It should be sufficient to check a single random pair of `x` and `y` since there are no special branches among the tested values. Additionally, it would be good to check the special cases, i.e., `xexpx` with `x = -Inf` and `xexpy` with `y = -Inf`. I don't think this is possible with ChainRulesTestUtils/FiniteDifferences though since the non-finite values will mess with finite differencing. So instead probably one just has to check the output of `frule` and `rrule` manually.
```suggestion
    x = randn()
    y = randn()
    # Test regular branch
    test_scalar(xexpx, x)
    test_frule(xexpy, x, y)
    test_rrule(xexpy, x, y)

    # Test special cases:
    # manual checks since FiniteDifferences/ChainRulesTestUtils can't differentiate around `-Inf`
    @test @inferred(frule((NoTangent(), randn()), xexpx, -Inf)) === (0.0, 0.0)
    Ω, back = @inferred(rrule(xexpx, -Inf))
    @test Ω === 0.0
    @test back(randn()) === (NoTangent(), 0.0)
    @test @inferred(frule((NoTangent(), randn(), randn()), xexpy, x, -Inf)) === (0.0, 0.0)
    Ω, back = @inferred(rrule(xexpy, x, -Inf))
    @test Ω === 0.0
    @test back(randn()) === (NoTangent(), 0.0, 0.0)
```